lib/ngx/resp.md

```diff
@@ -87,6 +88,14 @@ status with an optional reason. The `reason` should be a string.
 
 [Back to TOC](#table-of-contents)
 
+bypass_if_checks
+------------------
+**syntax:** *ngx_resp.bypass_if_checks()*
```

Comment:
Need a blank line after it. And don't forget to add the `[Back to TOC](#table-of-contents)`

Reply:
done